### ISSUE TITLE
Feature request - custom (or different) names for subcommands and arguments in help panel

### ISSUE THREAD

**Issue by vince-marco** [scallop/scallop#135]
I really like that scallop manages my help screen, but would like to clean up a couple of things.  First, I'd really like to change the "Subcommand:" prompt to "Command" at the top level.  This just produces confusion for users in the help panel.  Also would like to change "trailing arguments:" to "arguments:".

**Comment by Rogach** [comment]
Yes, I can add that. Will something like the following syntax be okay?

```scala
object Conf extends ScallopConf(args) {
  subcommandHelpPrefix = "Command:"
  trailingArgsHelpHeader = "arguments:"
}
```

**Comment by vince-marco** [comment]
That looks great.  One other thing (that is related).  You currently put a newline after options and before arguments, but no newline between commands.  Now with this approach I could put in a newline between commands with subcommandHelpPrefix = "\nCommand:".  But I can't take out the newline between options and arguments.  If you took out the newline there, then users could add it back if they wanted via --   trailingArgsHelpHeader = "\narguments:".   This would allow me to bunch up commands in the help, or (if desired) separate options from arguments (the current default) help layout.  Only thing is it wouldn't be the default.  Your call, of course.

**Comment by Rogach** [comment]
As a second thought, creating lots of arbitrarily named parameters on ScallopConf does not look as a best idea (you and I would be the only two persons capable of understanding what they mean).

Maybe it will be better to move help formatting into overridable method on ScallopConf? This way, you would be able to tweak help generation exactly the way you want, without wading through lots of cryptic options.

**Comment by vince-marco** [comment]
I'm fine with that.  Sounds like a great idea, in fact.  And I know I'm pushing into the finer aspects of the framework.  Thanks for the support.  Awesome framework!!

**Comment by Rogach** [comment]
Sorry for taking so long to implement this!

Implemented proposed change in version `3.0.0`, which should sync to Maven Central within an hour.

Custom help formatting is achieved by setting `helpFormatter`:
```scala
new ScallopConf(args) {
  helpFormatter = new ScallopHelpFormatter {
    // method overrides here
  }
}
```

Examples of how your requests can be implemented using this approach: https://github.com/scallop/scallop/blob/develop/jvm/src/test/scala/HelpTest.scala#L420

**Comment by Rogach** [comment]
Closing the issue. Feel free to reopen if this approach won't work for you.
